doc/classes/Control.xml

```diff
@@ -383,7 +383,7 @@
 		<method name="get_global_rect" qualifiers="const">
 			<return type="Rect2" />
 			<description>
-				Returns the position and size of the control relative to the top-left corner of the screen. See [member position] and [member size].
+				Returns the position and size of the control relative to the top-left corner of the [CanvasLayer]. See [member global_position] and [member size].
```

Comment:
A CanvasLayer does not have a top-left corner. I would probably write "Returns the position and size of the control relative to the [CanvasLayer]".